netebpfext/net_ebpf_ext_hook_provider.h

```diff
@@ -124,16 +124,16 @@ typedef struct _net_ebpf_extension_hook_provider_parameters
  * @param[in] attach_callback Pointer to callback function to be invoked when a client attaches.
  * @param[in] detach_callback Pointer to callback function to be invoked when a client detaches.
  * @param[in] custom_data (Optional) Opaque pointer to hook-specific custom data.
- * @param[in,out] provider_context Pointer to the provider context being registered.
+ * @param[in, out] provider_context Pointer to the provider context being registered.
  *
  * @retval STATUS_SUCCESS Operation succeeded.
  * @retval STATUS_NO_MEMORY Not enough memory to allocate resources.
  */
 NTSTATUS
 net_ebpf_extension_hook_provider_register(
     _In_ const net_ebpf_extension_hook_provider_parameters_t* parameters,
-    _In_ net_ebpf_extension_hook_on_client_attach attach_callback,
-    _In_ net_ebpf_extension_hook_on_client_detach detach_callback,
+    _In_ const net_ebpf_extension_hook_on_client_attach attach_callback,
+    _In_ const net_ebpf_extension_hook_on_client_detach detach_callback,
```

Comment:
```suggestion
    _In_ net_ebpf_extension_hook_on_client_attach attach_callback,
    _In_ net_ebpf_extension_hook_on_client_detach detach_callback,
```
I don't think const provides any value here since it's just making the argument immutable